src/ai/contexts.cpp

```diff
@@ -772,13 +771,6 @@ const moves_map& readonly_context_impl::get_possible_moves() const
 }
 
 
-const std::vector<unit_ptr>& readonly_context_impl::get_recall_list() const
-{
-	///@todo 1.9: check for (level_["disallow_recall"]))
-	return current_team().recall_list().recall_list_; //TODO: Refactor ai so that friend of ai context is not required of recall_list_manager at this line
```

Comment:
This TODO comment makes me wonder if removing this allows also removing a friend declaration somewhere else?

Reply:
It was possible and is now done in f8871d9